voice/methods/calls/getRecordings.md

```diff
@@ -0,0 +1,91 @@
+{% method %}
+
+## Retrieve Account Recordings
+Retrieve information about your 1000 least recent recordings.
```

Comment:
I don't know if I like saying your 1000 least recent recordings here.  You can get more than that if we add minStartTime and maxStartTime parameters.  It would probably be better to say that it returns a max of 1000 recordings, sorted by startTime from oldest to newest.  Or something like that.

Reply:
Agree, copypasta 🍝